src/test/common/terminals/environmentActivationProviders/terminalActivation.testvirtualenvs.ts

```diff
@@ -132,7 +132,7 @@ suite('Activation of Environments in Terminal', () => {
     ): Promise<string> {
         const terminal = vscode.window.createTerminal();
         await sleep(consoleInitWaitMs);
-        terminal.sendText(`python ${pythonFile} ${logFile}`, true);
+        terminal.sendText(`python ${pythonFile.toCommandArgument()} ${logFile.toCommandArgument()}`, true);
```

Comment:
Simple quoting works too
```suggestion
        terminal.sendText(`python '${pythonFile}' '${logFile}'`, true);
```

Reply:
`toCommandArgument()` is better. When we do the shell work it will be easier to find these.